### PR TITLE
Unwrap optional roadClass

### PR DESCRIPTION
**Pull request by bsudekum**
Followup to https://github.com/mapbox/MapboxDirections.swift/pull/180

We were sending across: `\Optional(motorway)`.

/cc @1ec5 @JThramer

### CODE REVIEW

File: MapboxDirections/MBRouteOptions.swift [hunk 1]
```diff
@@ -442,10 +442,10 @@ open class RouteOptions: NSObject, NSSecureCoding, NSCopying{
         
         if !roadClassesToAvoid.isEmpty {
             let allRoadClasses = roadClassesToAvoid.description.components(separatedBy: ",")
-            let firstRoadClass = String(describing: allRoadClasses.first)
             if allRoadClasses.count > 1 {
                 assert(false, "`roadClassesToAvoid` only accepts one `RoadClasses`.")
             }
+            let firstRoadClass = String(describing: allRoadClasses.first!)
```
Comment (1ec5) [inline MapboxDirections/MBRouteOptions.swift]: You can avoid the `!` by replacing `!roadClassesToAvoid.isEmpty` above with `let roadClass = allRoadClasses.first`.

**Comment by bsudekum** [comment]
@1ec5 fixed up. Couldn't do you comment because `roadClassesToAvoid` is non-optional.

File: MapboxDirections/MBRouteOptions.swift [hunk 2]
```diff
@@ -442,12 +442,12 @@ open class RouteOptions: NSObject, NSSecureCoding, NSCopying{
...
         if !roadClassesToAvoid.isEmpty {
             let allRoadClasses = roadClassesToAvoid.description.components(separatedBy: ",")
-            let firstRoadClass = String(describing: allRoadClasses.first)
             if allRoadClasses.count > 1 {
                 assert(false, "`roadClassesToAvoid` only accepts one `RoadClasses`.")
             }
-            params.append(URLQueryItem(name: "exclude", value: firstRoadClass))
-
+            if let firstRoadClass = allRoadClasses.first {
+                params.append(URLQueryItem(name: "exclude", value: String(describing: firstRoadClass)))
```
Comment (1ec5) [inline MapboxDirections/MBRouteOptions.swift]: Is it necessary to essentially convert this string into a string?

Comment (bsudekum) [reply MapboxDirections/MBRouteOptions.swift]: Guess not.

**Comment by bsudekum** [comment]
@1ec5 fixed

**Pull request merged**